apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/MulitpleDatabaseFirestoreFixture.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Http;
+using Google.Cloud.ClientTesting;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests;
+
+[CollectionDefinition(nameof(MulitpleFirestoreDbFixture))]
+public class MulitpleFirestoreDbFixture : CloudProjectFixtureBase, ICollectionFixture<MulitpleFirestoreDbFixture>
+{
+    private const string RestApiUrlBase = "https://firestore.googleapis.com/v1";
+    // Credentials scope, To view and manage your Google Cloud Datastore data.

```

Comment:
I'd probably delete the comment - or maybe rewrite it as "Scope used for the REST API to create databases"